aom/src/main/java/com/nedap/archie/adlparser/treewalkers/CComplexObjectParser.java

```diff
@@ -19,10 +20,12 @@
 public class CComplexObjectParser extends BaseTreeWalker {
 
     private final PrimitivesConstraintParser primitivesConstraintParser;
+    private final MetaModels metaModels;
```

Comment:
This field is not actually used. Is it still needed?

Reply:
Hmm, you're right. It will be needed for some future features though (see regular primitive objects in the issues). So it's fine to leave it here I think, so it's easier to develop that next feature :)